src/components/messages/messages-container-header/MessagesContainerHeader.vue

```diff
@@ -0,0 +1,51 @@
+<template>
```

Comment:
The correct naming here is `conversation` instead `messages`
The path should be `src/components/conversation/conversation-container-header/ConversationContainerHaeder.vue`
But better is: `src/components/conversation/conversation-receiver-header/ConversationReceiverHaeder.vue`
Because we introduce now the concept of a `receiver` by Luc which is more clear